src/Adafruit_MAX31865.h

```diff
@@ -52,6 +59,7 @@
 #include "WProgram.h"
 #endif
 
+// NOTE: Modify Adafruit_SPIDevice.h:36 = #if (defined(__AVR__) || defined(TEENSYDUINO)) && !defined(ARDUINO_CI)
```

Comment:
Can you say a bit more about this? As I recall it has something to do with `BUSIO_USE_FAST_PINIO`. Is this related to [`arduino_ci#193`](https://github.com/Arduino-CI/arduino_ci/issues/193)? If so, add a link and create a "tracking" issue in this project referencing the other project so we are reminded to come back to this when the other is resolved.